roles/openshift_node/meta/main.yml

```diff
@@ -13,9 +13,15 @@ galaxy_info:
   - cloud
 dependencies:
 - role: openshift_node_facts
+  when: not (openshift_node_upgrade_in_progress | default(False))
```

Comment:
I'm not crazy about adding booleans here, but it seems like the least-effort way to combine these roles.

After the roles are combined, I propose we refactor the meta depends out of both.

Reply:
Agreed.